homework7/7.1/7.1/Calculator.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace Task7_1;
+
+/// <summary>
+/// Класс, выполняющий операции с двумя числами
+/// </summary>
+public class Calculator
+{
+    /// <summary>
+    /// Первое операнд
+    /// </summary>
+    private double? numberOne;
+
+    /// <summary>
+    /// Второй операнд
+    /// </summary>
+    private double? numberTwo;
+
+    /// <summary>
+    /// Операция
+    /// </summary>
+    private char? operation;
+
+    /// <summary>
+    /// Добавление операнда
+    /// Если первый операнд существует, то
+    /// добавляет или изменяет второй.
+    /// Если первого операнда нет, то 
+    /// добавляет первый операнд
```

Comment:
```suggestion
    /// Если первый операнд существует, то добавляет или изменяет второй. Если первого операнда нет, то добавляет первый операнд
```
Не то чтобы Вы в чём-то неправы, но "стихотворное" оформление комментария немного смущает :)